modules/core/arrow-core/src/main/kotlin/arrow/core/internal/atomic.kt

```diff
@@ -22,8 +22,6 @@ class AtomicRefW<A>(a: A) {
   fun compareAndSet(expect: A, update: A) = atomicRef.compareAndSet(expect, update)
 
   fun lazySet(a: A) = atomicRef.lazySet(a)
-
-  override fun toString(): String = atomicRef.toString()
```

Comment:
Why is this one not working for JDK 7? could we print `atomicRef.value` instead of removing?

Reply:
It seems that they may been doing something behind the scenes, because I checked the source code and `toString` should delegate on the actual value, but the stack trace says differently:

> java.lang.NoSuchMethodError: No virtual method toString(Ljava/lang/Object;)Ljava/lang/String; in class Ljava/util/concurrent/atomic/AtomicIntegerFieldUpdater; or its super classes (declaration of 'java.util.concurrent.atomic.AtomicIntegerFieldUpdater' appears in /system/framework/core-libart.jar)
    

Using `value.toString` seems to work 👍